osu.Game/Overlays/Changelog/ChangelogHeader.cs

```diff
@@ -73,29 +67,40 @@ private void showBuild(ValueChangedEvent<APIChangelogBuild> e)
 
         protected override Drawable CreateBackground() => new OverlayHeaderBackground(@"Headers/changelog");
 
-        protected override Drawable CreateContent() => new Container
+        protected override Drawable CreateContent()
         {
-            RelativeSizeAxes = Axes.X,
-            AutoSizeAxes = Axes.Y,
-            Children = new Drawable[]
+            var content = new Container
             {
-                streamsBackground = new Box
-                {
-                    RelativeSizeAxes = Axes.Both
-                },
-                new Container
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Children = new Drawable[]
                 {
-                    RelativeSizeAxes = Axes.X,
-                    AutoSizeAxes = Axes.Y,
-                    Padding = new MarginPadding
+                    streamsBackground = new Box
                     {
-                        Horizontal = 65,
-                        Vertical = 20
+                        RelativeSizeAxes = Axes.Both
                     },
-                    Child = Streams = new ChangelogUpdateStreamControl()
+                    new Container
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Padding = new MarginPadding
+                        {
+                            Horizontal = 65,
+                            Vertical = 20
+                        },
+                        Child = Streams = new ChangelogUpdateStreamControl()
+                    }
                 }
-            }
-        };
+            };
+
+            Streams.Current.ValueChanged += e =>
```

Comment:
This feels like a huge hit in readability. Make a local bindable and use `BindTarget` and revert this code or something.

Reply:
Fair point.